publisher/publisher.go

```diff
@@ -369,3 +391,37 @@ func CreateTestingSignedSCT(req []string, k *ecdsa.PrivateKey, precert bool, tim
 	jsonSCT, _ := json.Marshal(jsonSCTObj)
 	return jsonSCT
 }
+
+func (pub *Impl) ProbeLogs() {
+	wg := new(sync.WaitGroup)
+	for _, log := range pub.ctLogsCache.LogURIs() {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			c := http.Client{
```

Comment:
Should this `http.Client` also have the timeouts we added recently in https://github.com/letsencrypt/boulder/commit/69f4f666b6d18198e45d1b347196549bbebf0964 ? `TLSHandshakeTimeout` in particular stands out as potentially relevant here.

Reply:
Probably, that said this is meant to just mirror what the underlying CT client uses currently. If we want to add those timeouts to that we should probably do that in a separate PR.

Reply:
That sounds good to me. To be clear you're suggesting a separate PR against certificate-transparency-go to add the timeouts, not a separate Boulder PR? Or both?

Reply:
We initialize the HTTP client in boulder, so we only need to fix it here.